core/src/mindustry/ai/types/MinerAI.java

```diff
@@ -63,7 +63,7 @@ protected void updateMovement(){
                 return;
             }
 
-            if(unit.within(core, unit.type.range)){
+            if(unit.within(core, unit.type.miningRange)){
```

Comment:
The changes on this line and the one below don't seem necessary - mining range should not affect core deposit & circling radius.